src/main/java/org/synyx/urlaubsverwaltung/application/statistics/ApplicationForLeaveStatistics.java

```diff
@@ -17,21 +19,27 @@
 public class ApplicationForLeaveStatistics {
 
     private final Person person;
+    private final PersonBasedata personBasedata;
 
     private final Map<VacationType, BigDecimal> waitingVacationDays = new HashMap<>();
     private final Map<VacationType, BigDecimal> allowedVacationDays = new HashMap<>();
 
     private BigDecimal leftVacationDays = ZERO;
     private Duration leftOvertime = Duration.ZERO;
 
-    ApplicationForLeaveStatistics(Person person) {
+    ApplicationForLeaveStatistics(Person person, PersonBasedata personBasedata) {
```

Comment:
Is `ApplicationForLeaveStatistics`the domain or the dto class? If it is the dto we should not add a domain class here.

Reply:
It's currently not well-defined. There is only `ApplicationForLeaveStatistic` in the statistics domain, which is build by the Service based on other domain types (like `Person`).

Reply:
Should we add a DTO here and use the `ApplicationForLeaveStatistic` as domain class? Then we can define the `ApplicationForLeaveStatisticsDto` exactly how we need it.

Reply:
We will add Dto layer